packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/NotOwnedSandboxInfo/Summary.tsx

```diff
@@ -92,9 +94,9 @@ export const Summary = () => {
               <Stack gap={2} align="center" paddingX={2}>
                 <Avatar user={author} />
                 <Stack direction="vertical">
-                  <Text variant={team ? 'body' : 'muted'} block>
+                  <Link variant={team ? 'body' : 'muted'} block>
```

Comment:
Link without an a?

Reply:
Yeah, the whole row is wrapped in a Link too, so this even propagates upwards and works